app/services/data_migrations/delete_uuidless_courses.rb

```diff
@@ -0,0 +1,15 @@
+module DataMigrations
+  class DeleteUuidlessCourses
+    TIMESTAMP = 20210601155313
+    MANUAL_RUN = false
+
+    def change
+      courses_without_uuid = Course.where(uuid: nil)
```

Comment:
Maybe add a line here to `raise` if `courses_without_uuid.flat_map(&:application_choices).any?`

Reply:
Just in case